docs/advanced/general-privacy.md

```diff
@@ -0,0 +1,63 @@
+# General information on Privacy
+
+## About transaction graphs
+
+It’s important to understand, that transactions are pseudonymous, but not anonymous. That means, a transaction graph can be built up, and there exists companies that are doing that work. If at some point one of our transactions is connected to our real world identity, it becomes easy to make assumptions based on that, for example that other transactions also belong to us, or that a specific unspent transaction belongs to us (that could give information about our balance). 
+By  checking the status of our transaction on a blockchain explorer, our IP address can be connected to one of our wallet addresses, this could create a starting point for a transaction graph. A possible solution for this is to use blockchain explorer only over Tor.
+
+## Transaction broadcasting
+
+Broadcasting the transaction can reveal that the transaction you are broadcasting belongs to you. A solution for this is to broadcast your transaction through a website, over Tor. For example, [https://explorer.dcrdata.org/decodetx](https://explorer.dcrdata.org/decodetx)
```

Comment:
Broadcasting **a** transaction* 


also, rather than duplicating the URL using \[]\(), we can just use <>.

`<https://explorer.dcrdata.org/decodetx>`